actors/runtime/runtime.go

```diff
@@ -129,7 +129,7 @@ type Syscalls interface {
 	// the "parent grinding fault", in which case it must be the sibling of h1 (same parent tipset) and one of the
 	// blocks in the parent of h2 (i.e. h2's grandparent).
 	// Returns nil and an error if the headers don't prove a fault.
-	VerifyConsensusFault(h1, h2, extra []byte, earliest abi.ChainEpoch) (*ConsensusFault, error)
+	VerifyConsensusFault(h1, h2, extra []byte, earliest abi.ChainEpoch, reporter addr.Address) (*abi.ConsensusFault, error)
```

Comment:
ooooooh, nasty. Reporter has been added here just to get it into the ConsensusFault object so that object can then be used as a parameter to a method? Please don't do that.
- if you re-organize as I suggested, this won't be necessary at all
- even if you don't, the SlashConsensusFault method should take a parameters struct with exactly the info it needs, which is _not_ the full ConsensusFault object but would include the reporter.